test/ex_unit_properties_test.exs

```diff
@@ -72,6 +74,52 @@ defmodule ExUnitPropertiesTest do
       assert Agent.get(counter, & &1) == 10
     end
 
+    property "runs for the specified number of milliseconds" do
+      {:ok, counter} = Agent.start_link(fn -> 0 end)
+
+      # I'm explicitly passing max_runs: :not_set to override the project-wide
+      # config in the `mix.exs` file of `max_runs: 100`.
+      check all :ok <- :ok, max_runs: :not_set, max_run_time: 250 do
```

Comment:
I think `nil` is a better default for `max_runs`, as I used in the snippet of code above.

Reply:
I started with `nil` as the default, but the problem I ran into there was this line: https://github.com/whatyouhide/stream_data/pull/72/files#diff-be707432787f16f5351a8cc945bdb9cbR387

When we passed in `nil` as an option it would be falsey, and so we default to the application environment setting, which meant I couldn't expressly override the application configuration for this test.

We could change that line to use `Keyword.has_key?/2` to set the default so that we can override with `nil` by passing it in if you think that's preferable.